.github/workflows/test-linux.yml

```diff
@@ -68,6 +77,12 @@ jobs:
             PYTHON_VERSION: '3.8'
     timeout-minutes: 90
     steps:
+      - name: Setup Remote SSH Connection
+        if: env.ENABLE_SSH == 'true'
+        uses: mxschmitt/action-tmate@v3
+        timeout-minutes: 10
```

Comment:
10 minutes seems too little to me, especially if we're going to check a failing test on CIs. 45 (as before) or 60 minutes would be better for me.

Reply:
No problem. I'll change it to 60. FYI, the timeout starts at the end of the workflow (action shutdown), not at this step.